Apps/Laboratory/src/Models/Covid19Model.cs

```diff
@@ -18,14 +18,15 @@ namespace HealthGateway.Laboratory.Models
     using System;
     using System.Collections.Generic;
     using System.Text.Json.Serialization;
+    using HealthGateway.Laboratory.Models.PHSA;
 
     /// <summary>
-    /// An instance of a Laboratory Model.
+    /// An instance of a COVID-19 Model.
     /// </summary>
-    public class LaboratoryModel
+    public class Covid19Model
```

Comment:
It looks like you've renamed **Covid19Model** to **PhsaCovid19Order** rather than just **Covid19Order**.

The model that PHSA returns should be PhsaCovid19Order (stored in Models/PHSA).  In the service, we convert that to a Covid19Order (stored in Models) that we return to the front-end.